src/data/faq_de.json

```diff
@@ -376,6 +379,216 @@
+                                "active": false,
+                                "textblock": [
+                                    "Die Warnung anderer mit der neuen Funktion \"Tests erfassen und andere warnen\" führt gar nicht zur Anzeige eines positiven Testergebnisses in der Corona-Warn-App.",
+                                    "Für Tests, die wie bisher in der App registriert werden, gilt nach wie vor: Tests stellen in der Diagnostik von Infektionskrankheiten immer nur eine Momentaufnahme dar, sie sind daher nicht unbegrenzt lange gültig. Nach 72 Stunden wird daher ein registrierter Test in der Corona-Warn-App als „veraltet“ grau dargestellt."
+                                ]
+                            },
+                            {
+                                "title": "Stoppt nach dem Teilen des positiven Ergebnisses meine Risikoermittlung? Wie lange?",
+                                "anchor": "warn_without_tan_risk_assessment",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, die Risikoermittlung und das Contact Tracing Protokoll werden nicht gestoppt. Die Warnung anderer mit den neuen Funktion \"Tests erfassen und andere warnen\" hat keine Auswirkungen auf die Risikoermittlung.",
+                                    "Wenn Sie sich trotz positiven Tests in der Öffentlichkeit bewegen, achten Sie bitte auf die Abstands- und Hygieneregeln und tragen Sie, wenn möglich, eine Maske zur Verringerung einer Infektionsgefahr für andere."
+                                ]
+                            },
+                            {
+                                "title": "Unterscheidet die Risikokalkulation bei einer Warnung auf meinem Smartphone je nach Test-Art  von der positiv getesteten Person?",
+                                "anchor": "warn_without_tan_risk_assessment_different",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, das Verfahren zur Kalkulation des Infektionsrisikos wurde nicht geändert. Unabhängig von der Art des Tests zum Nachweis einer Infektion gelten dieselben Paramater bei der Berechnung des Risikos für andere."
+                                ]
+                            },
+                            {
+                                "title": "Verschwindet eine Warnung auf Basis der neuen Funktion von selbst von der App, oder muss ich sie selbständig löschen?",
+                                "anchor": "warn_without_tan_warning",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, das ist nicht nötig. Nach einer Zeitspanne von max. 10 Tagen wird die Warnung in der App automatisch zurückgesetzt, weil dann auch das Zeitfenster für die Risikowarnungen erschöpft ist."

```

Comment:
In my opinion this entry is both hard to understand and could be misleading.
Reason:
1. When I first read this, I thought this is an entry for the user who used the SRS feature to warn others & now wants to know when others are not warned **anymore**!
2. After reading it a second time, I understood that this is actually a question the user who **received** a warning would ask. However, as the user can't differentiate between a normal warning and a SRS warning, I doubt that this question will be asked, and it definitely won't be a **frequently** asked question.

--> Conclusion: Remove this entry or re-write it to be more clear!

Reply:
See also comments for English language entry.

Reply:
Deleted in https://github.com/corona-warn-app/cwa-website/pull/3322/commits/ff61d079ed080cd3b21a3a6996709f3b3f04ad26, thanks!

Reply:
This conversation can be marked as resolved.